ivy/functional/frontends/numpy/linalg/solving_equations_and_inverting_matrices.py

```diff
@@ -18,3 +18,11 @@ def inv(a):
 # pinv
 def pinv(a, rtol=1e-15, hermitian=False):
     return ivy.pinv(a, rtol)
+    
+    
+# tensorinv
+def tensorinv(a, b, reverse=False, ind=2):
```

Comment:
tensorinv takes only `a` and `ind` as arguments. As seen here: https://numpy.org/doc/stable/reference/generated/numpy.linalg.tensorinv.html

Reply:
I saw that but if I remove the b and reverse arguments then the lint test fails.